subscription-manager.spec

```diff
@@ -359,18 +438,21 @@ rm -rf %{buildroot}
     %dir %{_sysconfdir}/yum.repos.d
 %endif
 
-%dir %{python_sitelib}/rhsmlib/candlepin
-%dir %{python_sitelib}/rhsmlib/compat
-%dir %{python_sitelib}/rhsmlib/dbus
-%dir %{python_sitelib}/rhsmlib/dbus/facts
-%dir %{python_sitelib}/rhsmlib/dbus/objects
-%dir %{python_sitelib}/rhsmlib/facts
-%dir %{python_sitelib}/rhsmlib/services
-%dir %{python_sitelib}/subscription_manager-%{version}-*.egg-info
-%dir %{python_sitelib}/subscription_manager/api
-%dir %{python_sitelib}/subscription_manager/branding
-%dir %{python_sitelib}/subscription_manager/model
-%dir %{python_sitelib}/subscription_manager/plugin
+%dir %{python_sitearch}/rhsmlib/candlepin
```

Comment:
Are we actually required to move to sitearch?  The rhsm subpackage will need to go there, but the subscription-manager stuff doesn't have to does it?  I don't oppose the move, but I just want to make sure we considered both options.

Reply:
Unfortunately, if we build it as one distribution (one `setup.py`, then everything ends up in `sitearch` due to logic that [puts any distribution that has an extension into a platform-specific directory](https://github.com/python/cpython/blob/b7cbfe49e39a7bbd7da20b937735a8a60bbf1872/Lib/distutils/command/install.py#L336).

Reply:
Interesting!  No big deal.  Just thought I'd ask.  We'll just have a really big RPMDiff section to waive :)

Reply:
👍 for having all modules in `/usr/lib64/pythonX.Y/site-packages`